packages/components/src/DataViewer/Text/SimpleTextKeyValue/SimpleTextKeyValue.component.js

```diff
@@ -103,6 +108,11 @@ export default function SimpleTextKeyValue({
 				<span className={classNames(theme['tc-simple-text-key'], 'tc-simple-text-key')}>
 					{formattedKey}
 					{separator}
+					{displayTypes && schema && value && (
+						<span className={classNames(theme['tc-simple-text-type'], 'tc-simple-text-type')}>
```

Comment:
Do we need to add theses classes as they are declared in the html container above ?

Reply:
It is not the same classes

Reply:
Yep 👀 I will open my eyes better the next time